radio/src/gui/480x272/model_setup.cpp

```diff
@@ -76,6 +76,8 @@ enum MenuModelSetupItems {
   ITEM_MODEL_SETUP_INTERNAL_MODULE_PXX2_MODEL_NUM,
   ITEM_MODEL_SETUP_INTERNAL_MODULE_OPTIONS,
   ITEM_MODEL_SETUP_INTERNAL_MODULE_AUTOBIND,
+  ITEM_MODEL_SETUP_INTERNAL_MODULE_DISABLE_TELEM,
```

Comment:
It should be inside MULTIMODULE ifdef (as done on external module), no?

Reply:
It's now intentionally controllable from the radio so you can disable it for just the models that need it.

Reply:
Bertrand was right, fixes it